perllib/FixMyStreet/App/Controller/Waste/Bulky.pm

```diff
@@ -135,6 +135,8 @@ sub view : Private {
     $c->stash->{template} = 'waste/bulky/summary.html';
 
 
+    $c->cobrand->bulky_set_reconstructed_property_data($p, $c->stash->{property});
```

Comment:
Is this needed because summary is constructing the payment from the items in the summary template? That template doing that feels wrong to me if it's being used to view an existing booking - we shouldn't reconstruct the cost in case the payment items have changed in the interim, we should use the stored `payment` data on the record if it's present, as that was the actual amount.

I think if we do that, hopefully all of this commit isn't really needed because it won't need to know if it were domestic/trade.

Reply:
Makes sense, my bad. Have still kept the part of the commit that does the domestic/trade identification based on the services rather than via property type. Will make the view collection page show the charged price as part of the payment work.